build_tools/gradle/build_tflite_android_library.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Cross-compile IREE's TFLite Java Bindings with Gradle and CMake. Produces an
+# iree-tflite-bindings-debug.aar and an iree-tflite-bindings-release.aar
+# library under bindings/tflite/java/output. Designed for CI, but can be run
+# manually from within the https://github.com/google/iree repo.
```

Comment:
Seems strange to mention a specific github repo. Git is distributed, and there is no way to "run a script from within a hosted github repo", you only run scripts from actual checkouts.

```suggestion
# library under bindings/tflite/java/output. Designed for CI, but can be run
# manually.
```